log4j-api/src/main/java/org/apache/logging/log4j/util/PropertySource.java

```diff
@@ -81,16 +81,22 @@ default boolean containsProperty(String key) {
     }
 
     /**
-     * Comparator for ordering PropertySource instances by priority.
-     *
-     * @since 2.10.0
-     */
+	 * Comparator for ordering PropertySource instances by priority.
+	 * Specifically, this Comparator will sort by Priority (int) in
+	 * <i>reverse</i> numerical order because a source with priority -100 has a
+	 * <i>higher</i> priority (can override values from) a source with priority
+	 * 100.
+	 *
+	 * @since 2.10.0
+	 */
```

Comment:
You may want to reformat your code.

Reply:
oops! I fixed that once but I must have gone back and changed something and forgot to disable format-on-save. I noticed some imports got unnecessarily changed too. Should be all fixed now.